packages/ui/src/components/organisms/ReviewModal/styles.scss

```
@@ -0,0 +1,50 @@
+[data-fs-review-modal] {
+  // --------------------------------------------------------
+  // Design Tokens for Review Modal
+  // --------------------------------------------------------
+
+  // Default properties
+  --fs-review-modal-width                   : 100%;
+  --fs-review-modal-max-width               : none;
+  --fs-modal-max-height-tablet                  : min(90vh, 44rem); // 704px
+  --fs-modal-body-padding                       : 0 var(--fs-spacing-4) var(--fs-spacing-3);
+
+  // --------------------------------------------------------
+  // Structural Styles
+  // --------------------------------------------------------
+
+  display: flex;
+  flex-direction: column;
+
+  [data-fs-review-modal-body] {
+    flex: 1;
+    overflow-y: auto;
+  }
+
+  @include media("<tablet") {
+    --fs-modal-position-top                     : 0;
+    --fs-modal-position-right                   : 0;
+    --fs-modal-position-left                    : 0;
```

```suggestion
    --fs-modal-position-right                   : var(--fs-modal-position-top);
    --fs-modal-position-left                    : var(--fs-modal-position-right);
```

maybe this work for this case?